includes/class-modal-checkout.php

```diff
@@ -273,6 +270,19 @@ function ( $item ) {
 				\wc_get_page_permalink( 'checkout' )
 			);
 
+			// Get data to send for this purchase.
+			$checkout_button_metadata = [
+				'currency'   => \get_woocommerce_currency(),
+				'amount'     => $price,
+				'product_id' => $product_id,
+				'referer'    => $referer,
```

Comment:
Nit: in general, unless we're literally calling a PHP or WP function that intentionally spells it `referer`, we should use the spelling `referrer` for consistency. This includes when using defining variable names as well as the param name in the payload.

```suggestion
				'referrer'    => $referer,
```

Reply:
D'oh! I remembered [this fix](https://github.com/Automattic/newspack-plugin/pull/2917) backwards 🤦‍♀️ Thank you!